SwrveSDK/src/google/java/com/swrve/sdk/Swrve.java

```diff
@@ -237,69 +235,22 @@ protected boolean checkPlayStoreSpecificArguments(String purchaseData, String re
         return true;
     }
 
+    @Override
     public void onResume(Activity ctx) {
         super.onResume(ctx);
         try {
             if (config.isPushEnabled()) {
-                if (activityContext != null) {
-                    Activity relatedActivity = activityContext.get();
-                    if (relatedActivity != null) {
-                        processIntent(relatedActivity.getIntent());
-                    }
-                }
                 if (qaUser != null) {
-                    qaUser.bindToServices();
+                    qaUser.bindToServices(); // todo is this needed?
```

Comment:
I don't think so no, moreover, it will be improved once we do the QA Log Improvements